src/number-input/number.component.ts

```diff
@@ -246,7 +246,7 @@ export class Number implements ControlValueAccessor {
 	emitChangeEvent(): void {
 		let event = new NumberChange();
 		event.source = this;
-		event.value = this.value;
+		event.value = +this.value;
```

Comment:
It may be more explicit/consistent to just call `Number` ... like:
```suggestion
		event.value = Number(this.value);
```

Reply:
I think I tried to use that, but `Number` was colliding with the component name. I will give it a try tomorrow.

Reply:
Ah shoot, that's right ... This is a touch more complicated, but can we rename the component to `NumberComponent` and add an `export { NumberComponent as Number };` to the end of the component and the end of the module? We'll have to fix the naming with v11

Reply:
Does `Number(this.value);` work now? If not we can take this for now